pkg/query/filter_test.go

```diff
@@ -30,10 +30,11 @@ func testPackages() map[string]*spdx.Package {
 		p := spdx.NewPackage()
 		p.ID = s
 		p.Name = fmt.Sprintf("gcr.io/puerco-chainguard/images/%s:v9.0.2-buster", s)
-		dg := "sha256:4ed64c2e0857ad21c38b98345ebb5edb01791a0a10b0e9e3d9ddde185cdbd31a"
+		dg := "sha256:4ed64c2e0857ad21c38b98345ebb5edb01791a0a10b0e9e3d9ddde185cdbd31a" //nolint:gosec
```

Comment:
```suggestion
		dg := "sha256:4ed64c2e0857ad21c38b98345ebb5edb01791a0a10b0e9e3d9ddde185cdbd31a" //nolint: gosec
```

Reply:
@cpanato ah it looks like a difference between behavior my local golangci-lint which was `v1.54.2` showed `go run mage.go verify` that calls `golangci-lint` and 1.55.2 on prow. 

I've force pushed as automation has already updated release-utils so I've removed that change in my branch. Happy to squash the changes so the adding/removing of the `nolint:gosec` goes away 

 I wonder if the makefile should call `mage.RunGolangCILint("", true)` to avoid such environment inconsistencies similarly to how zeitgeist is force installed.

On main and with golangci-lint 1.54.2 I get

```
golangci-lint --version && golangci-lint run
golangci-lint has version 1.54.2 built with go1.21.0 from 411e0bbb on 2023-08-21T12:04:32Z
pkg/query/filter_test.go:33:3: G101: Potential hardcoded credentials (gosec)
		dg := "sha256:4ed64c2e0857ad21c38b98345ebb5edb01791a0a10b0e9e3d9ddde185cdbd31a"
		^
pkg/query/filter_test.go:36:4: G101: Potential hardcoded credentials (gosec)
			dg = "sha256:c0d8e30ad4f13b5f26794264fe057c488c72a5112978b1c24f3940dfaf69368a"
			^
pkg/spdx/spdx_unit_test.go:302:2: G101: Potential hardcoded credentials (gosec)
	singleRef := "registry.k8s.io/kube-apiserver@sha256:1a61b61491042e2b1e659c4d57d426d01d9467fb381404bff029be4d00ead519"
```

vs my branch

```
golangci-lint --version && golangci-lint run
golangci-lint has version 1.54.2 built with go1.21.0 from 411e0bbb on 2023-08-21T12:04:32Z
```

Then updating golangci-lint the G101 errors are gone but I then see the `//nolint:gosec` is unused for linter "gosec` errors seen in prow. Note the suggested space doesn't fix the error.